apps/ai-content-generator/src/locations/ConfigScreen.jsx

```diff
@@ -0,0 +1,106 @@
+
+  return (
+    <Flex flexDirection="column" className={css({ margin: '80px', maxWidth: '800px' })}>
+      <Form>
+        <FormControl isRequired>
+          <FormControl.Label>OpenAI API key</FormControl.Label>
+          {editing ? (
+            <TextInput
+              value={apiKey}
+              type="text"
+              name="apikey"
+              placeholder="sk-...4svb"
+              onChange={(e) => {
+                setApiKey(e.target.value);
+                setParameters({ ...parameters, key: e.target.value });
+              }}
+              onBlur={(e) => setEditing(false)}
+            />
+          ) : (
+            <TextInput
+              isReadOnly={true}
+              value={parameters.key?.replace(/.(?=.{4,}$)/g, '*')}
+              type="text"
+              name="apikey"
+              placeholder="sk-...4svb"
+              onClick={() => setEditing(true)}
+            />
+          )}
+          <FormControl.HelpText>Provide your API key registered with OpenAI</FormControl.HelpText>
+          {!parameters?.key && (
+            <FormControl.ValidationMessage>
+              You can visit the website of OpenAI and register for an API key. Provide the API key
+              here.
+            </FormControl.ValidationMessage>
+          )}
+        </FormControl>
+        <FormControl>
+          <FormControl.Label>Describe your profile</FormControl.Label>
+          <Textarea
+            rows={15}
+            value={parameters.profile || ''}
+            type="text"
+            name="profile"
+            placeholder="Example: we sell online clothing for the conscious consumer. We care about the planet and make sure our products are made in a sustainable way. Our customers are young and care about the environment. We are progressive, creative, fun and also serious about our business."
+            onChange={(e) => setParameters({ ...parameters, profile: e.target.value })}
+          />
+          <FormControl.HelpText>
+            AI Content Generator can take a certain profile in which it will generate content. If
+            you leave this profile empty, content suggestions will most likely not be relevant.
+            Write the profile as if you are describing your online business.
+          </FormControl.HelpText>
+        </FormControl>
+        <FormControl>
+          <FormControl.Label>Machine Learning Model</FormControl.Label>
+          <Select
+            value={parameters.model || 'gpt-3.5-turbo'}
+            onChange={(e) => setParameters({ ...parameters, model: e.target.value })}>
+            <Select.Option value="gpt-3.5-turbo">gpt-3.5-turbo (recommended)</Select.Option>
+            <Select.Option value="gpt-3.5-turbo-0301">gpt-3.5-turbo-0301</Select.Option>
+            <Select.Option value="davinci">text-davinci-003</Select.Option>
+          </Select>
+          <FormControl.HelpText>
+            Different models generate different results! gpt-3.5-turbo is the latest and most
```

Comment:
Can you adjust this to GPT-4?

Reply:
yeah, i'll take davinci out while we're at it to. it will remove divergent code paths

Reply:
pushed an update


https://user-images.githubusercontent.com/492573/229806915-1cf6dc2d-c4e9-4d29-a913-16acf4902734.mov